portfolio/src/main/webapp/script.js

```diff
@@ -83,17 +83,16 @@ function deleteComment(comment) {
 
 
 function UserLogin(){
```

Comment:
This piece `loggedIn => loggedIn.text()` tells JS that you want to read the response as text.  Since you now have a json response, you can maybe do `loggedIn => loggedIn.json()`.  You also then wouldn't need the `String()` and `trim()` pieces.

Reply:
See how you have the `then()` for `loadComments()`